### PR TITLE
Add radio type and fixed mime type support and theme support

### PR DESCRIPTION
**Pull request by AIIX**
#### Description
- Adds support for fixed mime types that Skills can send to open that specific player type ["type/video", "type/audio", "type/radio"]
- Adds support for getting theme from play() metadata and forwarding it to playback skill if available
- Adds support to display radio type

#### Type of PR
- [ ] Bugfix
- [x] Feature implementation
- [ ] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements

#### Testing
For Testing: https://github.com/MycroftAI/skill-npr-news/pull/120
Requires: https://github.com/MycroftAI/skill-playback-control/pull/49

#### CLA
- [x] Yes

### CODE REVIEW

File: plugin_playback_guiplayer/__init__.py [hunk 1]
```diff
@@ -64,28 +65,46 @@ def _get_track(self, track_data):
             track = track_data[0]
             mime = track_data[1]
             mime = mime.split('/')
+            theme = track_data[2]
```
Comment (krisgesling) [inline plugin_playback_guiplayer/__init__.py]: Should we check the length of track_data before trying to access this item that potentially isn't there? 

This should mean existing Skills won't break it.

**Review by krisgesling — COMMENTED** [review]
Hey, happy to add this in its current form, potentially with the theme item optional, so that any Skills that want to use it can do so and Skills still using the `(track, mime)` format won't break it.

What do you think?

File: plugin_playback_guiplayer/__init__.py (hunk 1 above)
Comment (AIIX) [reply plugin_playback_guiplayer/__init__.py]: should be fixed in latest commit

Comment (krisgesling) [reply plugin_playback_guiplayer/__init__.py]: Great, thanks!

**Pull request merged**